lib/publify/api.rb

```diff
@@ -0,0 +1,14 @@
+require 'net/http'
+
+module Publify
+  class API
+    def self.latest_links(limit)
+      uri = URI("http://#{PUBLIFY_HOSTNAME}/articles.json")
+      response = Net::HTTP.get uri
```

Comment:
Looking at the [docs](http://ruby-doc.org/stdlib-2.1.1/libdoc/net/http/rdoc/Net/HTTP.html#class-Net::HTTP-label-GET), seems this could be written as:

```ruby
response = Net::HTTP.get(PUBLIFY_HOSTNAME, '/articles.json')
```